.github/workflows/build.yml

```diff
@@ -1,17 +1,14 @@
-# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
-
 name: build
 
 on: [push, pull_request]
 
 jobs:
-  build:
+
+  lint:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8]
-
+        python-version: [3.7]
```

Comment:
The matrix is unnecessary.

Reply:
`matrix` in lint is deleted. But it is kept in `build_**` for potential future usage.